### ISSUE TITLE
Interacting with a faction camp's bulletin board while dragging a vehicle makes the vehicle glitch out

### ISSUE THREAD

**Issue by MildlySereneFox** [CleverRaven/Cataclysm-DDA#58469]
### Describe the bug

If you grab onto a drag-able, one-tile vehicle (like a shopping cart, a self-made one-tile travois or similar) to drag it with you, then interact with the bulletin board of a faction camp without letting go of it and quit out of the faction camp menu, the vehicle will glitch out.

In general, it will stay where it was when you move. You can grab it again, but moving will give the message "Can't find grabbed object." and let go of it.
When you move far enough away (a couple of tiles), the vehicle sometimes suddenly "jumps" by one tile and becomes drag-able again. Saving and reloading also fixes it.
While testing, I've also once randomly managed to duplicate a shopping cart this way - until one of them suddenly vanished and the other "jumped" to another tile.

### Steps to reproduce

1. Create a new, empty faction camp out in a field.
2. Spawn in an (undamaged) shopping cart.
3. Grab onto the shopping cart.
4. Interact with the bulletin board to open the camp menu.
5. Quit out of the faction camp menu.
6. Try to grab and drag the shopping cart again.

### Expected behavior

That you either are still dragging the vehicle after closing the camp menu, or that you properly let go of it when you use the bulletin board and can simply grab it again.

### Versions and configuration

- OS: Linux
    - OS Version: LSB Version: n/a; Distributor ID: openSUSE; Description: openSUSE Leap 15.3; Release: 15.3; Codename: n/a; 
- Game Version: f6be61e [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda]
]

**Comment by PatrikLundell** [comment]
/confirmed

I grabbed a welding cart in experimental downloaded and compiled yesterday and tried to interact with the bulletin board (using an existing save).
The first time the cart ceased to be grabbed by still jumped one tile after the PC when he moved away. The second time the grabbed vehicle was simply released, a few additional attempts eventually resulted in a duplicate cart that disappeared when the dragged cart was released (and the grabbing of it wasn't released when the board was exited).
Thus, weird semi random things seem to be possible.

I tried an MC a couple of times, and those cases I consistently had the PC move away from the MC and when two steps away a message of "Can't find grabbed object" appeared and the MC never moved (until grabbed again).

I agree that the behavior should be consistent (and not weird either).
